ghost/posts-service/lib/PostsService.js

```diff
@@ -526,7 +526,8 @@ class PostsService {
                 'published_at',
                 'published_by',
                 'canonical_url',
-                'count__clicks'
+                'count__clicks',
+                'newsletter_id'
```

Comment:
When we add another field to the post would it automatically be included in the copied version? Maybe worth reconsidering allowlist vs blocklist approach?

Reply:
💯 - the pattern of remembering to omit fields vs allowing them when needed just leads to a world of pain

Reply:
Alrighty then, i swapped it out to explicitly define the fields to be copied over